Gameplay.java

```diff
@@ -0,0 +1,226 @@
+package brickBreaker;
+
+import java.awt.Color;
+import java.awt.Font;
+import java.awt.Graphics;
+import java.awt.Graphics2D;
+import java.awt.Rectangle;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.awt.event.KeyEvent;
+import java.awt.event.KeyListener;
+import javax.swing.Timer;
+
+import javax.swing.JPanel;
+
+public class Gameplay extends JPanel implements KeyListener, ActionListener {
+    private boolean play = false;
+    private int score = 0;
+     
+    private int totalBricks = 21;
+    
+    private Timer timer;
+    private int delay = 8;
+    
+    private int playerX = 300;
+    
+    private int ballposX =120;
+    private int ballposY = 350;
+    private int ballXdir = -1;
+    private int ballYdir = -2;
+    
+    private MapGenerator map;
+    
+    
+    public Gameplay()
+    {
+    	map = new MapGenerator(5,8);
```

Comment:
Wouldn't it be better if this `MapGenerator` instance was named something else? Because `MapGenerator` has a `map` member too

Reply:
Can we name it wall generator instead of map generator?

Reply:
The problem here is that `MapGenerator` has a `map` member so here, naming this instance `map` would be confusing as explained in the comment below:

> `map.map` looks a bit confusing so it'd be good if the name was changed